### PR TITLE
Added Fluiditype

### PR DESCRIPTION
**Pull request by sboriskirov**
Adding a site build with Gatsby, which is about fluid typography

## Description

Added another example for the Showcase, a typography library with a website built with Gatsby

### CODE REVIEW

**Comment by laurieontech** [comment]
Thanks for the PR! I noticed that the README doesn't include any instructions for running the site. Could you add those? Then we should be able to get this merged.

**Comment by sboriskirov** [comment]
> Thanks for the PR! I noticed that the README doesn't include any instructions for running the site. Could you add those? Then we should be able to get this merged.

@laurieontech Thank you for the comment, but I only added my website in the sites.yml which README should I update?

**Comment by laurieontech** [comment]
Totally understand. Sorry for not being clearer! The [github link](https://github.com/sboriskirov/fluiditype) you included for the site has a README.

**Comment by sboriskirov** [comment]
> Totally understand. Sorry for not being clearer! The [github link](https://github.com/sboriskirov/fluiditype) you included for the site has a README.



> Totally understand. Sorry for not being clearer! The [github link](https://github.com/sboriskirov/fluiditype) you included for the site has a README.

Thank you for the explanation, that part is done you can review it here [https://github.com/sboriskirov/fluiditype/blob/master/README.md](url)

**Comment by laurieontech** [comment]
Ah ok, I misunderstood. Since the github link is for the package rather than the site you're sharing we should remove it. The goal of the source_url is to share Gatsby source code when/where appropriate. I'll go ahead and remove it and merge this in. Thanks again for your contributions!

File: docs/sites.yml [hunk 1]
```diff
@@ -10140,6 +10140,18 @@
     - Web Development
   built_by: Jason Zheng
   built_by_url: https://github.com/bilafish
+- title: Fluiditype
+  url: https://www.fluiditype.com/
+  main_url: https://www.fluiditype.com/
+  source_url: https://github.com/sboriskirov/fluiditype
```
Comment (laurieontech) [inline docs/sites.yml:10146]:
```suggestion
```

**Comment by sboriskirov** [comment]
> Ah ok, I misunderstood. Since the github link is for the package rather than the site you're sharing we should remove it. The goal of the source_url is to share Gatsby source code when/where appropriate. I'll go ahead and remove it and merge this in. Thanks again for your contributions!

Sorry I did not make things clear it was to help out, the site itself is build with Gatsby but the concept behind the website is hidden into getting the .css file. Thank you for taking the time.

**Pull request merged**